api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.7.x.xml

```diff
@@ -169,5 +169,13 @@
 			referencedTableName="obs"
 			referencedColumnNames="obs_id"/>
 	</changeSet>
+
+	<changeSet id="TRUNK-6280-2024-11-01" author="manojll">
+		<preConditions  onFail="MARK_RAN">
```

Comment:
Let us try as much as we can to avoid mixing database changes JPA annotations migration.

Reply:
@ManojLL let's create a separate ticket and a PR for this.

Reply:
@wikumChamith sure